smash/core/model/_standardize.py

```diff
@@ -533,17 +533,26 @@ def _standardize_rr_parameters_value(
     if not isinstance(value, (int, float, np.ndarray)):
         raise TypeError("value argument must be of Numeric type (int, float) or np.ndarray")
 
-    arr = np.array(value, ndmin=1)
-    low, upp = FEASIBLE_RR_PARAMETERS[key]
-    low_arr = np.min(arr)
-    upp_arr = np.max(arr)
+    arr = np.array(value, ndmin=1, dtype=np.float32)
 
-    if isinstance(value, np.ndarray) and value.shape != model.mesh.flwdir.shape and value.size != 1:
+    if arr.shape != model.mesh.flwdir.shape and arr.size != 1:
```

Comment:
Pourquoi pas faire ça
```suggestion
    if arr.size != 1:
        # Check all values
        mask = np.ones(arr.shape, dtype=bool)
        
    elif arr.shape == model.mesh.flwdir.shape:
        # Do not check if a value is inside the feasible domain outside of active cells
        mask = model.mesh.active_cell == 1
        
    else:
```